packages/activation-notice/src/app/components/step1.js

```diff
@@ -65,6 +64,20 @@ function Step1() {
     trackClick(evt, 'open_demo_story');
   }, []);
 
+  const translatedString = createInterpolateElement(
+    __('Read the<a>Get Started story</a>', 'web-stories'),
```

Comment:
```suggestion
    __('Read the <a>Get Started story</a>', 'web-stories'),
```

Without the `br` I suppose we should add a space everywhere instead.